### PR TITLE
HV-1070 Making build more friendly towards Java 9

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@gsmet Another quick one :)

File: distribution/pom.xml [hunk 1]
```diff
@@ -95,6 +95,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
+                    <additionalJOption>-addmods java.xml.bind</additionalJOption>
```
Comment (gsmet) [inline distribution/pom.xml]: This breaks the build on Java 8. It should be added in a profile (or removed as you claim we do not support building the distribution with Java 9).

Thanks!

Comment (gunnarmorling) [inline distribution/pom.xml]: Doh, yes. Pushed an update.

**Comment by gsmet** [comment]
Rebased and merged! Thanks!

**Pull request closed without merging**